src/contracts/Yieldy.sol

```diff
@@ -207,6 +207,7 @@ contract Yieldy is
         address _to,
         uint256 _value
     ) public override returns (bool) {
+        require(_value <= balanceOf(_from), "Transfer greater than balance");
```

Comment:
nitpick - but might increase readability to have both this require statement and the next in same direction..

```
require(balanceOf(_from) >= _ value, "Transfer greater than balance");
require(_allowances[_from][msg.sender] >= _value, "Allowance too low");